content/beginner/190_efs/cleaning.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Clean Up"
+date: 2019-04-09T00:00:00-03:00
+weight: 15
+draft: false
+---
+
+### Cleaning up
+To delete the resources used in this chapter: 
+```
+cd ~/environment/efs
+kubectl delete -f efs-reader.yaml
+kubectl delete -f efs-writer.yaml
```

Comment:
This is repeat and fails; previous step already deleted the reader and writer pods. Please remove.

Reply:
Got it. Removed the delete commands